packages/tracking/src/integrations/onetrust.ts

```diff
@@ -32,7 +32,7 @@ export const initializeOneTrust = async ({
       scope.dataLayer ??= [];
       scope.dataLayer.push({ event: 'OneTrustGroupsUpdated' });
       resolve();
-      script.parentNode?.removeChild(script);
+      // script.parentNode?.removeChild(script);
```

Comment:
Did we want to remove this or just comment for now?

Reply:
meant to keep this actually, good catch